src/scripts/helpers/devLogger.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Development Logging Utilities
+ *
```

Comment:
I am a little confused about the use case for these functions. I think we already have Morgan for logging but can we not use different log levels (info, debug, warning, error etc.) to keep the logs clean and set where we want logging to output (e.g. console or file)?

Reply:
Morgan logs HTTP requests/responses and devLogger logs application's internal logic. They serve different purposes.